README.md

```diff
@@ -23,17 +23,15 @@ The routes are of course fake but you can get a grasp of what lout looks like gi
 Lout depends on vision and inert, make sure you register them with hapi.
 
 ```javascript
-var Hapi = require('hapi');
-var server = new Hapi.Server();
+const Hapi = require('hapi');
 
-server.connection({ port: 80 });
+const server = new Hapi.Server({ port: 80 });
 
-server.register([require('vision'), require('inert'), { register: require('lout') }], function(err) {
-});
+server.register([require('vision'), require('inert'), { register: require('lout') }]);
```

Comment:
If I run this snippet, I end up the following result: 
![image](https://user-images.githubusercontent.com/1080566/34324067-1aa4fffc-e831-11e7-8879-3383de280394.png)

I believe if you update this line to:

````javascript
await server.register([require('vision'), require('inert'), require('lout')])
````
the example works.

Reply:
Agreed with that, the simpler form is better now.